### PR TITLE
update bonus epsiode with minor correction

### PR DESCRIPTION
**Pull request by amanmdesai**
<details>
<summary><strong>Instructions</strong></summary>

Thanks for contributing! :heart:

Keep in mind that **lesson maintainers are volunteers** and it may take them some time to respond to your contribution.
To ensure that the right people get notified, you can tag some of the last contributors with `@githubname`.

Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum. If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact the HSF training convenors ([contacts here](https://hepsoftwarefoundation.org/workinggroups/training.html)).

You may delete these instructions from your pull request.

\- HSF Training
</details>

### CODE REVIEW

**Comment by amanmdesai** [comment]
Hi @klieret, these are just some minor modifications for bonus episode related mostly to formatting.

File: _episodes/08-bonus-episode.md [hunk 1]
```diff
@@ -59,26 +59,25 @@ From: ubuntu:20.04
 As we see, several packages are installed and this installation is done over several layers.
 
 
-> ### Bonus question: Why is installation over several layers advantageous?
-{: .challenge}
-
-> ### Solution:
+> ## Bonus question: Why is installation over several layers advantageous?
+> ## Solution:
```
Comment (klieret) [inline _episodes/08-bonus-episode.md]: Hmm, are you sure you don't need double `>` here? As in 
`> > ## Solution` etc.?

Comment (klieret) [reply _episodes/08-bonus-episode.md]: You can probably find a similar example somewhere in another lesson

File: _episodes/08-bonus-episode.md [hunk 2]
```diff
@@ -115,7 +114,7 @@ jobs:
            singularity push container.sif oras://ghcr.io/${GITHUB_REPOSITORY}:${tag}
 ```
 
-The above script is designed to build and publish any Singularity image with GitHub packages.
+The above script is designed to build and publish Singularity image with GitHub packages.
```
Comment (klieret) [inline _episodes/08-bonus-episode.md]: **a** Singularity image?

**Review by klieret — CHANGES_REQUESTED** [review]
I think the bonus question layout is probably not yet correct... Does that work locally?

File: _episodes/08-bonus-episode.md (hunk 2 above)
Comment (amanmdesai) [reply _episodes/08-bonus-episode.md]: Done

File: _episodes/08-bonus-episode.md (hunk 1 above)
Comment (amanmdesai) [reply _episodes/08-bonus-episode.md]: > I think the bonus question layout is probably not yet correct... Does that work locally?

It worked locally. I was using the Bonus question template here: https://hsf-training.github.io/hsf-training-matplotlib/05-mplhep/index.html 
But it seems that we need to update that template as well. (I will open a new PR in matplotlib lesson.)

The current commit should work.

Comment (amanmdesai) [reply _episodes/08-bonus-episode.md]: Thanks!

**Review by klieret — APPROVED** [review]
LGTM! Thanks a lot :)

**Pull request merged**